src/sh-utils.ts

```diff
@@ -49,42 +49,42 @@ class SHRotation {
 
         // band 1
         const sh1 = [
-            [ rot[4],-rot[7], rot[1] ],
-            [-rot[5], rot[8],-rot[2] ],
-            [ rot[3],-rot[6], rot[0] ]
+            [rot[4], -rot[7], rot[1]],
+            [-rot[5], rot[8], -rot[2]],
+            [rot[3], -rot[6], rot[0]]
         ];
 
         // band 2
         const sh2 = [[
             kSqrt01_04 * ((sh1[2][2] * sh1[0][0] + sh1[2][0] * sh1[0][2]) + (sh1[0][2] * sh1[2][0] + sh1[0][0] * sh1[2][2])),
-                          (sh1[2][1] * sh1[0][0] + sh1[0][1] * sh1[2][0]),
+            (sh1[2][1] * sh1[0][0] + sh1[0][1] * sh1[2][0]),
```

Comment:
Not sure if you want this formatting to be lost.

Reply:
Yeah you're right. I've reverted whitespace and disabled rule in that file. Thanks!